---
title: "Iterative correction of Hi-C data reveals hallmarks of chromosome organization"
date: 2012-10-01
publishDate: 2021-04-27T17:01:57.499694Z
authors: ["Maxim Imakaev", "Geoffrey Fudenberg", "Rachel Patton McCord", "Natalia Naumova", "Anton Goloborodko", "Bryan R. Lajoie", "Job Dekker", "Leonid A. Mirny"]
publication_types: ["2"]
abstract: "Extracting biologically meaningful information from chromosomal interactions obtained with genome-wide chromosome conformation capture (3C) analyses requires the elimination of systematic biases. We present a computational pipeline that integrates a strategy to map sequencing reads with a data-driven method for iterative correction of biases, yielding genome-wide maps of relative contact probabilities. We validate this ICE (iterative correction and eigenvector decomposition) technique on published data obtained by the high-throughput 3C method Hi-C, and we demonstrate that eigenvector decomposition of the obtained maps provides insights into local chromatin states, global patterns of chromosomal interactions, and the conserved organization of human and mouse chromosomes."
featured: false
publication: "*Nature Methods*"
tags: ["Chromatin", "Chromosome Mapping", "Chromosomes", "Human", "High-Throughput Screening Assays", "Humans", "Nucleic Acid Conformation"]
doi: "10.1038/nmeth.2148"
---

